clang-tools-extra/clangd/CodeComplete.cpp

```diff
@@ -561,14 +561,32 @@ struct CodeCompletionBuilder {
   }
 
   std::string summarizeSnippet() const {
+
+    /// localize a ArgList depending on the config. If the config is unset we
+    /// will use our local (this) version, else use the one of the config
+    const Config::ArgumentListsOption ArgList =
```

Comment:
For consistency with our existing `Completion` option (`AllScopes`), let's do the `Config` lookup [here](https://searchfox.org/llvm/rev/87d56c59f52d033cd7c46d769338b9c47fea4929/clang-tools-extra/clangd/ClangdServer.cpp#453) and propagate it into `CodeCompleteOpts`.

That then gets propagated to `CodeCompletionBuilder::ArgumentLists` in the `CodeCompletionBuilder` constructor, and here we can just use the `ArgumentLists` member without doing anything further.